machida/machida.pony

```diff
@@ -406,7 +406,7 @@ class PyTCPEncoder is TCPSinkEncoder[PyData val]
 
   fun apply(data: PyData val, wb: Writer): Array[ByteSeq] val =>
     let byte_buffer = Machida.sink_encoder_encode(_sink_encoder, data.obj())
-    if not byte_buffer.is_null() and not Machida.is_py_none(byte_buffer) then
```

Comment:
Same as above, we still need to do the null check because if the encoder raises an exception then the `byte_buffer` will be null and we should fail.